test/functional/p2p_orphan_handling.py

```
@@ -0,0 +1,416 @@
+class OrphanHandlingTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.extra_args = [[]]
+
+    def create_parent_and_child(self):
+        """Create package with 1 parent and 1 child, normal fees (no cpfp)."""
+        parent = self.wallet.create_self_transfer()
+        child = self.wallet.create_self_transfer(utxo_to_spend=parent['new_utxo'])
+        return child["tx"].getwtxid(), child["tx"], parent["tx"]
+
+    def relay_transaction(self, peer, tx):
```

Ruff (0.12.2)

Missing return type annotation for public function `relay_transaction`

Add return type annotation: `None`

(ANN201)